### ISSUE TITLE
Fix r2lang plugins after radare2 refactoring of RBin and RAnal+RAsm merge

### ISSUE THREAD

**Issue by XVilka** [radare/radare2-bindings#197]
Since the API will change drastically, both implementation and documentation (in r2book) should be fixed.

**Comment by radare** [comment]
Whats this? A bug coming from the future?

**Comment by XVilka** [comment]
Yes, a note from me in the future for younger myself to prevent a major
catastrophe that happens in one of our alternative futures.

**Comment by radare** [comment]
+1

**Comment by radare** [comment]
Obviously no . Not even really started
